sdl_android/src/main/java/com/smartdevicelink/trace/Mime.java

```diff
@@ -18,48 +22,61 @@ public static String base64Encode(String str) {
 	}
 
 	public static String base64Encode(byte bytesToEncode[]) {
-		return base64Encode(bytesToEncode, 0, bytesToEncode.length);
+		if(bytesToEncode != null){
+			return base64Encode(bytesToEncode, 0, bytesToEncode.length);
+		}
+		return null;

```

Comment:
Needs documentation explaining what is returned, and if null is provided, null will be returned.